docker-files/vessel

```diff
@@ -155,9 +155,8 @@ if [ $# -gt 0 ]; then
         shift 1
         if [ "$EXEC" == "yes" ]; then
             $COMPOSE exec \
-                -u vessel \
                 app \
-                ./vendor/bin/phpunit "$@"
+                /var/www/html/vendor/bin/phpunit -c /var/www/html/phpunit.xml "$@"
```

Comment:
I found an issue with this way :(

when we run "./vessel test" it will override the env. vars and if the testing database is different from the env. vars it causes an issue